### PR TITLE
Add checks on Python and WORKSPACE in git hooks

### PR DESCRIPTION
**Pull request by FunkySayu**

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -133,6 +133,12 @@ new_git_repository(
 load("@io_bazel_rules_go//go:def.bzl", "new_go_repository")
 
 new_go_repository(
+    name = "com_github_oleiade_lane",
```
Comment (cassand) [inline WORKSPACE:144]: It seems this repository is added but not used

**Review by cassand — CHANGES_REQUESTED** [review]

File: WORKSPACE (hunk 1 above)
Comment (cassand) [reply WORKSPACE:144]: Just moved in fact, good to go

**Review by cassand — APPROVED** [review]

File: devutils/git-hooks/pre-commit [hunk 2]
```diff
@@ -26,6 +26,16 @@ PreCommit::GoChecks () {
 }
 
 
+# Rules checking Python code validity
+PreCommit::PythonChecks () {
+    local filepath="$1"
+
+    which pep8 > /dev/null || Log::Failure "pep8 not found. Install it with pip install pep8."
```
Comment (cassand) [inline devutils/git-hooks/pre-commit:33]: It may be useful to make a page with all the dependencies for each workflow

Comment (FunkySayu) [reply devutils/git-hooks/pre-commit:33]: What do you mean? A function checking if all dependencies are installed? Or a documentation page?

**Pull request merged**